README.md

```diff
@@ -69,4 +65,22 @@ npm run start:template
 
 ## How to deploy to an organization
 
-TODO:
+Token Request app has been deployed to Rinkeby at `token-request.open.aragonpm.eth`
```

Comment:
token-request-staging.open.aragonpm.eth?

Reply:
We will most likely have all apps deployed pretty soon with the regular name (without staging)